src/server/api/controllers/routes.controller.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+
+const HttpError = require("../lib/utils/http-error");
+const knex = require("../../config/db");
+
+// Get all routes
+const getroutes = async req => {
+  try {
+    return await knex("routes").select(
+      "routes.id as id",
+      "voyage_id",
+      "eta",
+      "max_wave_height",
+      "hfo",
+      "lsfo",
+      "total_cost",
+      "distance_over_ground",
+      "distance_through_water",
+      "avgspeed"
+    );
+  } catch (err) {
+    console.log(err);
+  }
+};
+
+// Create a route
+const createroute = async ({ body }) => {
+  const {
+    voyage_id,
+    eta,
+    max_wave_height,
+    hfo,
+    lsfo,
+    total_cost,
+    distance_over_ground,
+    distance_through_water,
+    avgspeed
+  } = body;
+  const route = await knex
+    .from("routes")
+    .select("*")
+    .where({
+      id: voyage_id
+    });
+  if (route.length !== 0) {
```

Comment:
Hi Abod, here I think it should be === instead !==

Reply:
No it's not. But you're right that there is a problem here. We cannot create more than one suggested route that is connected to a voyage and we need to be able to do that.